src/main/java/com/quirko/app/GameController.java

```diff
@@ -6,41 +6,66 @@
 import com.quirko.logic.events.InputEventListener;
 import com.quirko.logic.events.MoveEvent;
 
+
 public class GameController implements InputEventListener {
 
-    private Board board = new SimpleBoard(25, 10);
+    private int level1TargetStart = 70;

```

Comment:
final static LEVEL_TARGET_START = 70;

Reply:
Meaningful constant is created to hold integer.